content/blocks/unloader.hjson

```diff
@@ -1 +1,15 @@
+name: Fast Unloader
+description: Unloader but faster.
+type: Unloader
+itemCapacity: 10
+hasItems: true
+health: 140
+size: 1
+requirements: [
+  titanium/25
+  silicon/30
+  thorium/20
```

Comment:
fast unloader's building cost is original unloader's cost + 20 thorium

Reply:
Have you tested this? I'm pretty sure it either replaces the current unloaders, or doesn't work at all.

Reply:
tested in campaign, 100% works

Reply:
I have successfully tested this on both Android and Windows
You can test it your self

Reply:
> 
> 
> Have you tested this? I'm pretty sure it either replaces the current unloaders, or doesn't work at all.

it adds another unloader, not replacing it

Reply:
I think I uploaded with the wrong name

Reply:
Ah okay, that makes more sense